### PR TITLE
Commerce 2036 Account entities now support Filters and Sorts

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:stop

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
ci:test

**Comment by andrea-ale-sbarra** [comment]
hi @ethanbustad , I wonder why these changes don't require the baseLine task to be run, do you know why? I tried to run that task locally but new version of module was not found

**Comment by ethanbustad** [comment]
ci:test
tests should be fixed in upstream now

**Comment by ethanbustad** [comment]
@andrea-ale-sbarra , I think it's just because the versions of these modules have already been increased since the last release. The baseline script checks the current state against the previous release artifact (in this case, whatever was released in Commerce `2.0.6`). So if we've already had a `baseline` increase since `2.0.6`, then you won't see any further increases (at least at that level) until the next release. Obviously if we've only seen a minor version increase so far, and you introduce a major change, that'd still cause a baseline change. Does that make sense of why this PR (and the other one you pinged me on) didn't require any baseline changes?

**Review by ethanbustad — APPROVED** [review]

File: headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/odata/entity/v1_0/AccountEntityModel.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+/**
+ * @author Andrea Sbarra
+ */
+public class AccountEntityModel implements EntityModel {
+
+	public AccountEntityModel() {
+		_entityFieldsMap = Stream.of(
+			new StringEntityField(
+				"name", locale -> Field.getSortableFieldName(Field.NAME))
```
Comment (marco-leo) [inline headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/odata/entity/v1_0/AccountEntityModel.java:35]: @andrea-ale-sbarra add  account type please

**Pull request merged**